PyStemmusScope/global_data/global_data_selection.py

```diff
@@ -0,0 +1,106 @@
+"""Module containing the full global dataset collection."""
+from pathlib import Path
+from typing import Any
+from typing import Dict
+from typing import Tuple
+from typing import Union
+import numpy as np
+import pandas as pd
+import xarray as xr
+from PyStemmusScope import global_data as gd
+from PyStemmusScope import variable_conversion as vc
+
+
+def collect_datasets(
+    global_data_dir: Path,
+    latlon: Union[Tuple[int, int], Tuple[float, float]],
+    time_range: Tuple[np.datetime64, np.datetime64],
+    timestep: str,
```

Comment:
I would agree that it would reduce the duplication in the docstrings, however, I do like the transparency and simplicity of the current code. Also, if we pass around a dict, this would make e.g., type annotations, more difficult.

Reply:
Yep, type annotation is a good concern. Anyway, it is a trade-off between simplicity and transparency. Actually I don't have very strong opinion about it, as it is merely a design choice, both have pros and cons. Let's just keep it for now then.